test/functional/page_objects/common_page.js

```diff
@@ -79,23 +79,28 @@ export function CommonPageProvider({ getService, getPageObjects, getPageObject }
 
             if (loginPage && !wantedLoginPage) {
               log.debug(`Found loginPage username = ${config.get('servers.kibana.username')}`);
-              await PageObjects.shield.login(
+              return PageObjects.shield.login(
```

Comment:
What's the reason behind these changes? Do you need the checking to live in a promise chain?